stock_no_negative_move_line/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
@gabriel-grinspan The manifest has wrong format.

See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst